package.json

```diff
@@ -54,10 +54,12 @@
   "scripts": {
     "analyze": "NODE_ENV=production ANALYZE=1 node --max_old_space_size=4096 node_modules/.bin/webpack --config webpack.prod.config.js --progress --colors",
     "build": "webpack --config webpack.prod.config.js",
+    "build-dev": "webpack --config webpack.dev.config.js",
```

Comment:
why do we/you need this?

Reply:
I occasionally wish to test the build without bringing up the dev server.